tutorials/finetune_lora.md

```diff
@@ -22,8 +22,12 @@ The steps here only need to be done once:
    python scripts/prepare_alpaca.py
    ```
 
+For more information about dataset preparation, also see the [prepare_dataset.md](./prepare_dataset.md) tutorial.
+
 (See [this blog article](https://lightning.ai/blog/how-to-finetune-gpt-like-large-language-models-on-a-custom-dataset) for how to prepare and use custom datasets.)
```

Comment:
Does the new tutorial replace this link? The other tutorials don't have it

Reply:
By link do you mean the link to this new "prepare_dataset.md" file that is to be added in this PR? The other 2 tutorials, that is, finetune_full.md and finetune_adapter.md have it too.

Regarding the 

> (See [this blog article](https://lightning.ai/blog/how-to-finetune-gpt-like-large-language-models-on-a-custom-dataset) for how to prepare and use custom datasets.)

I haven't added it or touched it afaik. I think that was already there before. I think we can remove it by the way because it doesn't go into the steps for preparing a dataset but is more of a general Lit-GPT user guide.. I think it's probably easier for someone to just copy & edit one of the existing `prepare_*.py` files.

What do you think?

Reply:
I agree

Reply:
ok, removed it!